shell/platform/android/io/flutter/plugin/localization/LocalizationPlugin.java

```diff
@@ -136,4 +184,39 @@ public void sendLocalesToFlutter(@NonNull Configuration config) {
 
     localizationChannel.sendLocales(locales);
   }
+
+  @VisibleForTesting
+  public static Locale localeFromString(String localeString) {
```

Comment:
Is this overall logic repeated anywhere? I feel like I have implemented this before. I will see if I can find it (if it exists). Best to reuse code if appropriate.

Reply:
Ahh, it was on the framework side. Never mind!